guides/features/pipecat-flows.mdx

```diff
@@ -1,441 +1,267 @@
-    icon="shield"
-    href="https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/dynamic/insurance_gemini.py"
+    title="Restaurant Reservation"
+    icon="utensils"
+    href="https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/dynamic/restaurant_reservation.py"
   >
-    A dynamic flow using Google Gemini that adapts policy recommendations based
-    on user responses. Demonstrates runtime node creation and conditional paths.
+    A simple, practical dynamic flow example
   </Card>
 </CardGroup>
 
-<Note>
-  These examples are fully functional and can be run locally. Make sure you have
-  the required dependencies installed and API keys configured.
-</Note>
-
-## When to Use Static vs Dynamic Flows
-
-**Static Flows** are ideal when:
-
-- Conversation structure is known upfront
-- Paths follow predefined patterns
-- Flow can be fully configured in advance
-- Example: Customer service scripts, intake forms
-
-**Dynamic Flows** are better when:
-
-- Paths depend on external data
-- Flow structure needs runtime modification
-- Complex decision trees are involved
-- Example: Personalized recommendations, adaptive workflows
-
-# Installation
-
-If you're already using Pipecat:
-
-```bash
-pip install pipecat-ai-flows
-```
-
-If you're starting fresh:
-
-```bash
-# Basic installation
-pip install pipecat-ai-flows
-
-# Install Pipecat with specific LLM provider options:
-pip install "pipecat-ai[daily,openai,deepgram]"     # For OpenAI
-pip install "pipecat-ai[daily,anthropic,deepgram]"  # For Anthropic
-pip install "pipecat-ai[daily,google,deepgram]"     # For Google
-```
-
-💡 Want to design your flows visually? Try the [online Flow Editor](https://flows.pipecat.ai)
+## How do Pipecat and Pipecat Flows work together?
 
-# Core Concepts
+**Pipecat** defines the core capabilities of your bot. This includes the pipeline and processors which, at a minimum, enable your bot to:
 
-## Designing Conversation Flows
+- Receive audio from a user
+- Transcribe the user's input
+- Run an LLM completion
+- Convert the LLM response to audio
+- Send audio back to the user
 
-Functions in Pipecat Flows serve two key purposes:
+**Pipecat Flows** complements Pipecat's core functionality by managing the conversation's context and tools. This is separate from the core pipeline, allowing you to separate the conversation logic from the core pipeline mechanics.
 
-- Processing data (likely by interfacing with external systems and APIs)
-- Advancing the conversation to the next node
+## When to Use Pipecat Flows?
 
-Each function can do one or both.
+Pipecat Flows is best suited for use cases where:
 
-LLMs decide when to run each function, via their function calling (or tool calling) mechanism.
+- You require precise control over how a conversation progresses
+- Your bot needs to handle a complex task that can be decomposed into discrete steps
+- You're trying to improve the accuracy of your LLM's response or tool use
```

Comment:
Not sure I grok this 3rd point. Can you explain?

Reply:
Ah, you're referring to cases where you might otherwise have a huge prompt and many tools registered at once. I think I understand. The follow sentence makes it clearer.

Reply:
Yes! Reads great now